### PR TITLE
Add Echelon (ECH) to frontier

### PR DESCRIPTION
**Pull request by echelonfoundation**

### CODE REVIEW

**Comment by sunnya97** [comment]
Hi, for now, could you please only add to frontier.

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 1]
```diff
@@ -1876,6 +1876,34 @@
...
+      "display": "echelon",
+      "symbol": "ECH",
+      "ibc": {
+        "source_channel": "channel-8",
+        "dst_channel": "channel-262",
+        "source_denom": "aechelon"
+      },
+      "logo_URIs": {
+        "png": "https://app.ech.network/logo.png",
+        "svg": "https://app.ech.network/echelon.svg"
```
Comment (sunnya97) [inline osmosis-1/osmosis-frontier.assetlist.json:1903]: Can you upload these images into an images subfolder in the chain registry

See example:  https://github.com/cosmos/chain-registry/tree/master/osmosis/images

Comment (echelonfoundation) [reply osmosis-1/osmosis-frontier.assetlist.json:1903]: Yes, will do!

File: osmosis-1/osmosis-1.assetlist.json [hunk 2]
```diff
@@ -1348,6 +1348,34 @@
...
+        "source_channel": "channel-8",
+        "dst_channel": "channel-262",
+        "source_denom": "aechelon"
+      },
+      "logo_URIs": {
+        "png": "https://app.ech.network/logo.png",
+        "svg": "https://app.ech.network/echelon.svg"
+      },
+      "coingecko_id": "echelon"
+    },
```
[the remark this answers is not in the document]
Comment (echelonfoundation) [reply osmosis-1/osmosis-1.assetlist.json:1378]: We do have $1,500+ in liquidity now, is there a set requirement on the required amount to be on osmosis-1? Thanks!

**Comment by echelonfoundation** [comment]
Changes made, thanks

**Pull request merged**